sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeBeforeApiSuccessTest.java

```diff
@@ -1498,4 +1501,40 @@ public boolean matches(Object argument) {
         };
         verify(httpClient).callAsync(argThat(urlArg), eq("GET"), eq(headers), any(HttpClient.CallTemplate.class), any(ServiceCallback.class));
     }
+
+    @Test
+    public void enqueueDistributionStartSessionLogAfterEnablingUpdates() throws Exception {
+
+        /* Setup mock. */
+        mockStatic(SessionContext.class);
+        SessionContext sessionContext = mock(SessionContext.class);
+        when(SessionContext.getInstance()).thenReturn(sessionContext);
+        SessionContext.SessionInfo sessionInfo = mock(SessionContext.SessionInfo.class);
+        when(sessionContext.getSessionAt(anyLong())).thenReturn(sessionInfo);
+        when(PreferencesStorage.getString(PREFERENCE_KEY_REQUEST_ID)).thenReturn("r");
+
+        /* Enable in-app updates. */
+        start();
+        Distribute.getInstance().storeRedirectionParameters("r", "g", null);
+
+        /* Verify the log was sent. */
+        verify(mChannel).enqueue(any(DistributionStartSessionLog.class), eq(Distribute.getInstance().getGroupName()));
+    }
+
+    @Test
+    public void DontEnqueueDistributionStartSessionLogIfNoSessionsWereLoggedBefore() throws Exception {
```

Comment:
camelCase for method names, not PascalCase

Reply:
Fixed. Also added a test to check for a `null` SID. Thank you.